docs/.vuepress/theme/components/SearchBox.vue

```diff
@@ -155,105 +172,82 @@ export default {
 
       // At first, search for the phrase in the API reference main categories
       for (let i = 0; i < pages.length; i++) {
-        if (resAPI.length >= maxAPI) {
```

Comment:
It has a negative performance impact. (I didn't test how large yet). If there aren't any objections, I would prefer to sort all pages through their priority during component or app initialization.  WDYT?

Reply:
I checked that and there are ~500-700 items to search (after that two loops + inner loops)? IMO it's nothing for JS.

Reply:
Ok, let's keep it as is. 
BTW. If any performance issues happen further, we should remove two loops first.